nats-base-client/jsstream_api.ts

```diff
@@ -85,9 +86,9 @@ export class StreamAPIImpl extends BaseApiClient implements StreamAPI {
     return new ListerImpl<StreamInfo>(subj, filter, this);
   }
 
-  async purge(name: string): Promise<PurgeResponse> {
+  async purge(name: string, opts?: PurgeOpts): Promise<PurgeResponse> {
```

Comment:
maybe worth detecting early when there are conflicting values - like you cant have seq and keep together afaik.

Reply:
I was hoping the compiler did it, but it looks like they are not there yet.

Reply:
LGTM